cpp/include/cudf/io/parquet.hpp

```diff
@@ -401,6 +401,9 @@ class parquet_writer_options {
   bool _write_timestamps_as_int96 = false;
   // Column chunks file path to be set in the raw output metadata
   std::string _column_chunks_file_path;
+  /// vector of precision values for decimal writing. Exactly one entry
+  /// per decimal column.
```

Comment:
In the chunked options this is mandatory if there are decimal columns. Want to make the docstring same?